script/state.py

```diff
@@ -1,5 +1,6 @@
 import json
 import psycopg2
+import togeojsontiles
```

Comment:
Same comment as in other PR, optimise the imports

```suggestion
import json

import psycopg2
import togeojsontiles
```

Reply:
all imports used in script (json used as i used json.dumps() -> converts a Python object into a json string )

Reply:
what i meant is you can `reorganise` the imports written in first 3 lines like that. Not specifically meant to replace any of the import. If you use IDE like Visual Studio or Pycharm, they have commands in them(`Optimise Import`) which helps us to reorganise the imports so that they can ge grouped accordingly(native libraries, third party libraries, user created libraries) like that

Reply:
Done